microsoft/PowerToys
# [VCM] add new option ToolbarHide After timeout and Startup action

## Summary of the Pull Request


## PR Checklist

- [x] **Closes:** #28677


## Detailed Description of the Pull Request / Additional comments
- Add new option ToolbarHide After timeout, allow always hide Toolbar no matter what state
- Add new behavior option startup action, allow Unmute/Mute when startup for privacy
- Fix Mute/Unmute logic, all controlled mic should be same state avoid manually change in Control Panel


## Validation Steps Performed
- Select behavior 'Mute' > Unmute all > Restart > All should muted
- Have multiple Microphone devices > Change system sound > Change Default device level to unmute, Other mute > PowerToys Toggle shortcut > All devices should mute
